kitchen.yml

```diff
@@ -22,6 +22,7 @@ suites:
     run_list:
       - recipe[osl-nginx::default]
       - recipe[osl-nginx::mon]
+    multiple_converge: 2
```

Comment:
You might as well remove this unless this specific suite works with `enforce_idempotency`

Reply:
oops, i thought when you said converge again above that you meant add a second converge sorry